sdk/tables/aztable/table_pagers.go

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Same thing here: there should not be a table_pagers.go; these constructs should be near the methods than require them.

Reply:
Will condense this, I did this to avoid bloat in the client and service_client files but I can see why this is better.